### PR TITLE
Fix activate script on Windows

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 package:
   name: conda-forge-build-setup
-  version: 1.0.1
+  version: 1.0.2
```
Comment (jakirkham) [inline recipe/meta.yaml]: Could we actually do 1.1.0? Was thinking the last number would represent pinning changes. The middle number represents package changes (bug fixes). The first number would represent breaking package changes.

Though we haven't actually agreed on these per se. It seems helpful to understand what has changed with this package if something goes wrong. This is one way we might do this, but would be open to other suggestions.

Comment (jjhelmus) [inline recipe/meta.yaml]: Sounds reasonable to me.

**Comment by jakirkham** [comment]
Other than the versioning comment, this LGTM. Thanks for fixing @jjhelmus. 😄

**Comment by jjhelmus** [comment]
Happy to help. The PDX airport has free wifi, next year someone should organize a sprint at the airport as part of PyCon.

**Pull request merged**